### ISSUE TITLE
With the liveSearch mode, the search button is displayed

### ISSUE THREAD

**Issue by Coroico** [Coroico/AdvSearch#14]
Even with liveSearch parameter equal to 1, the search button is displayed. It shouldn't be displayed.

**Comment by Coroico** [comment]
As workaround (and fix ?) modify the AdvSearchForm by replacing 

```
[[+liveSearch:isnot=`1`:then=`<input type="submit" id="[[+asId]]_advsea-submit"  name="sub" value="[[%advsearch.search? &namespace=`advsearch` &topic=`default`]]" />`]]
```

by:

```
[[+liveSearch:isnot=`1`:then=`<input type="submit" id="[[+asId]]_advsea-submit"  name="sub" value="[[%advsearch.search? &namespace=`advsearch` &topic=`default`]]" />`:else`=``]]
```

**Comment by Coroico** [comment]
Fixed with the commit 40c6e3a
